.github/workflows/merge-main.yml

```diff
@@ -150,7 +149,7 @@ jobs:
           overwrite: ${{ matrix.overwrite }}
           penetration_test: false
           parameters:
-            -p PROMOTE=ghcr.io/${{ github.repository }}/${{ matrix.name }}:test
+            -p PROMOTE=${{ github.repository }}/${{ matrix.name }}:test
```

Comment:
Sorry, hope this is my last question.
So this one should be `${{ github.repository }}/${{ matrix.name }}:${{ env.ZONE }}` as well, not `:test` right?

Reply:
Hi Derek,
Sorry, was I asking a wrong question?
So this one should be ${{ github.repository }}/${{ matrix.name }}:${{ env.ZONE }} as well, not :test right?

Reply:
Both PRs have been rolled together.  Hopefully that will explain everything!